src/ImageProcessor.Web/HttpModules/ImageProcessingModule.cs

```diff
@@ -214,12 +214,32 @@ public static void SetHeaders(HttpContext context, string responseType, string[]
 
             cache.SetExpires(DateTime.Now.ToUniversalTime().AddDays(maxDays));
             cache.SetMaxAge(new TimeSpan(maxDays, 0, 0, 0));
-            cache.SetRevalidation(HttpCacheRevalidation.AllCaches);
+            //cache.SetRevalidation(HttpCacheRevalidation.AllCaches);
```

Comment:
Just to make sure, should this be commented out? And if yes, why?
Make sure to remove it instead of commenting it out, this will leave people wondering forever: was this actually useful code? :-)